app/src/lib/stores/app-store.ts

```diff
@@ -6826,3 +6886,11 @@ function isLocalChangesOverwrittenError(error: Error): boolean {
     error.result.gitError === DugiteError.LocalChangesOverwritten
   )
 }
+
+function constrain(
+  value: IConstrainedValue | number,
+  min = -Infinity,
```

Comment:
TIL: You can specify infinity as a number 🤯

Reply:
It's the negative infinity that cracks me up! :smile: